src/constraints/ramping-and-unit-commitment.jl

```diff
@@ -0,0 +1,38 @@
+export add_ramping_constraints!
+
+"""
+add_ramping_constraints!(graph, )
+
+Adds the ramping constraints for producer and conversion assets where unit_commitment = true in assets_data
+"""
+function add_ramping_constraints!(model, graph, df_flows, flow, F, Auc, units_on, dataframes)
+    ## Expressions used by the ramping and unit commitment constraints
+
+    # - Flow that is above the minimum operating point of the asset
+    flow_above_min_oper_point = [
+        if row.from ∈ Auc               # Not sure if I should be working in an asset df or flow df
```

Comment:
@datejada I'm not sure if I should be working in a flow df or an asset df. I tried this, which I think uses the flow df and only uses flows out of assets that are in Auc.

Reply:
P.S. Don't worry about the extra arguments for the function, they're WIP.

Reply:
Good work here! You are moving at a good pace with this development :)
So, It should be the df flow. We want to limit the rate of change (i.e., ramping) in the outgoing flows of the asset. The column `from` (or something similar) in the df flows is the asset, and you need to check if it has the UC (or ramping) method (option) to create the expression.